doc/nrf/gs_modifying.rst

```diff
@@ -149,12 +149,6 @@ Hardware-related configuration
 
 The preprocessed devicetree sources are parsed by the :file:`zephyr/scripts/dts/gen_defines.py` script to generate a :file:`devicetree_unfixed.h` header file with preprocessor macros.
 
```

Comment:
This edit seems to have removed the following line from the final output: 

```suggestion

As a debugging aid, :file:`gen_defines.py` writes the final devicetree to :file:`zephyr.dts`.
```

If still relevant, it should be added directly in the file (without using ncs-include)